lib/open_food_network/order_cycle_management_report.rb

```diff
@@ -17,7 +17,7 @@ def header
     end
 
     def search
-      Spree::Order.complete.not_state(:canceled).distributed_by_user(@user).managed_by(@user).search(params[:q])
+      Spree::Order.complete.where("spree_orders.state != ?", :canceled).distributed_by_user(@user).managed_by(@user).search(params[:q])
```

Comment:
Bummer...we need this to be explicitly applied to spree_orders, otherwise we end up with this error when we try to scope by payment method: 'PG::Error: ERROR:  column reference "state" is ambiguous'.

Both orders and payments have a 'state' field. :(